docs/running-locust-docker.rst

```diff
@@ -0,0 +1,43 @@
+.. _running-locust-docker:
+
+=================================
+Running Locust with Docker
+=================================
+
+To keep things simple we provide a single Docker image that can run standalone, as a master, or as a slave. 
+
+
+Environment Variables
+---------------------------------------------
+
+- ``LOCUST_MODE``
+
+One of 'standalone', 'master', or 'slave'. Defaults to 'standalone'.
+
+- ``LOCUSTFILE_PATH``
+
+The path inside the container to the locustfile. Defaults to '/locustfile.py`
+
+- ``LOCUST_MASTER_HOST``
+
+The hostname of the master.
+
+- ``LOCUST_MASTER_PORT``
+
+The port used to communicate with the master. Defaults to 5557.
+
+
+Add your tests
+---------------------------------------------
+
+The easiest way to get your tests running is to build an image with your test file built in. Once you've 
+written your locustfile you can bake it into a Docker image with a simple ``Dockerfile``:
+
+```
+FROM locustio/locust
+ADD locustfile.py locustfile.py
```

Comment:
Maybe we should discourage the usage of `ADD` for the user's `locustfile.py` and document mounting a directory with the `locustfile.py` instead.